src/components/comments/VoteButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import cx from 'classnames'
+
+export default ({
+  isSignedIn,
+  numVotes,
+  userHasVoted,
+  onUpvote,
+  onUnvote
+}) => {
+  return (
+    <div
+      className={cx('vote-button', { active: userHasVoted, disabled: !isSignedIn })}
+      onClick={(e) => isSignedIn && (userHasVoted ? onUnvote() : onUpvote())}
```

Comment:
You can remove the `e` argument, and I'd also recommend fixing these parentheses (they kinda weirdly wrap random stuff, whereas it might be better to have them wrap the condition)